recon.sh

```diff
@@ -55,7 +58,7 @@ case $method in
     export JUGGLER_SIM_FILE=$sim_file
     export JUGGLER_REC_FILE=$rec_file
     export JUGGLER_IRT_AUXFILE=$aux_file
-    gaudirun.py juggler/JugPID/tests/options/irt.py
+    gaudirun.py irt.py
```

Comment:
Do you prefer to move the options file to `drich-dev`? I think we should keep it closer to `IRTAlgorithm`, at least for code preservation, but we will soon be moving away from Juggler anyway.